home/npc_movement.asm

```diff
@@ -50,5 +50,15 @@ RunNPCMovementScript::
 EndNPCMovementScript::
 	farjp _EndNPCMovementScript
 
-EmptyFunc2::
+DebugPressedOrHeldB::
+IF DEF(_DEBUG)
+	ld a, [wd732]
+	bit 1, a
```

Comment:
This is obviously the Field Debug flag. See `wDebugFlags` in pokecrystal.

Reply:
It's used for other things though, like whether play time is being counted, whether the Bicycle is force-enabled, what type of warp is going on, etc.

Reply:
This specific bit is?

Reply:
The byte is. It's one of many instances in pokered where named constants don't exist yet. I'd rather not create one just for that specific bit, but also don't want this to become a general improvement PR that identifies the other bits.